sage-ios/SAGE/SAGE/AddSchoolController.swift

```diff
@@ -41,15 +44,34 @@ class AddSchoolController: FormController {
         self.navigationController?.pushViewController(tableViewController, animated: true)
     }
     
+    func radiusButtonTapped() {
+        if self.location == nil {
+            self.showAlertControllerError("Please choose a location first.")
+        } else {
+            if self.location?.coordinate != nil {
+                self.radiusCenter = self.location!.coordinate
+            }
+            let viewController = AddSchoolRadiusViewController(center: self.radiusCenter!, radius: self.radius)
```

Comment:
is it possible to just pass in self.location.coordinate into the viewController? it doesnt seem like you need self.radiusCenter for anything else other than passing it as a param into the VC